_data/projects/Roc.yml

```diff
@@ -0,0 +1,12 @@
+name: Roc
+desc: Roc is a toolkit for real-time audio streaming over the network.
+site: https://roc-project.github.io/
+tags:
+- C++
+- Audio
+- DSP
+- streaming
+- networking
+upforgrabs:
+  name: help wanted
+  link: https://github.com/roc-project/roc/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@gavv Please change this to https://github.com/roc-project/roc/labels/help%20wanted

Reply:
Done.